### PR TITLE
fix(templates-ng): host context scope selectors

### PR DESCRIPTION
**Pull request by vchimev**
https://angular.io/guide/component-styles

### CODE REVIEW

**Comment by MartoYankov** [comment]
Should also update the master detail kinvey ng template.

**Comment by vchimev** [comment]
And verify the rest of the Angular templates. :)

**Comment by vchimev** [comment]
Update:
- [x] template-master-detail-ng
- [x] template-master-detail-kinvey-ng

File: packages/template-hello-world-ng/src/app/item/items.component.html [hunk 1]
```diff
@@ -26,8 +26,8 @@
 <GridLayout>
     <ListView [items]="items">
         <ng-template let-item="item">
-            <StackLayout>
-                <Label [nsRouterLink]="['/item', item.id]" [text]="item.name"></Label>
+            <StackLayout >
+                <Label [text]="item.name"  [nsRouterLink]="['/item', item.id]" backgroundColor="red"></Label>
```
Comment (manoldonev) [inline packages/template-hello-world-ng/src/app/item/items.component.html]: `backgroundColor="red"` was probably a debugging helper? This file seems like needing no changes -- as a best practice I tend to skim over all the files after I submit a PR to clean up such leftovers.

File: packages/template-tab-navigation-ng/src/app/home/home.component.html [hunk 2]
```diff
@@ -5,8 +5,8 @@
 <GridLayout>
     <ListView [items]="items">
         <ng-template let-item="item">
-            <StackLayout orientation="horizontal">
-                <Label [nsRouterLink]="['../item', item.id]" [text]="item.name"></Label>
+            <StackLayout orientation="horizontal" [nsRouterLink]="['../item', item.id]">
```
Comment (manoldonev) [inline packages/template-tab-navigation-ng/src/app/home/home.component.html]: Is there a specific reason for moving this one level up?

**Pull request merged**